wgpu-hal/src/dx12/shader_compilation.rs

```diff
@@ -145,6 +145,9 @@ mod dxc {
         let mut compile_flags = arrayvec::ArrayVec::<&str, 4>::new_const();
```

Comment:
You need to bump the capacity here too

Reply:
Ah, now the CI failure makes sense, thank you!

Reply:
Done.